lib/Reader/reader.cpp

```diff
@@ -8015,6 +8027,8 @@ void ReaderBase::msilToIR(void) {
     ++it;
   }
 
+  popDebugScope();
```

Comment:
This is being deleted as Pat pointed out that we never have more than one scope, so there's no point in having the stack.